File: Java/TP3-4/src/Regular.java
public class Regular extends Usuario {

	public Regular(String login) {
		super(login);
	}

	public void despedidaRegular() {
		System.out.println("Log-off " + super.getLogin());
	}

	public void descricaoRegular() {
		System.out.println("Pode consultar suas informações.");
	}

	public void dashboard() {
		System.out.println("Usuario: " + super.getLogin());
		System.out.println("Tipo: Regular");
		System.out.print("Permissoes: ");
		this.descricaoRegular();
	}
}

File: Kotlin/TPK3-4/src/Regular.kt
class Regular(login:String):Usuario(login) {
  fun despedidaRegular() {
    System.out.println("Log-off " + super.getLogin())
  }
  fun descricaoRegular() {
    println("Pode consultar suas informações.")
  }
  fun dashboard() {
    System.out.println("Usuario: " + super.getLogin())
    println("Tipo: Regular")
    print("Permissoes: ")
    this.descricaoRegular()
  }
}